scenario_player/main.py

```diff
@@ -370,6 +370,7 @@ def construct_rc_message(log_fpath) -> str:
         message = f":x: Error while running scenario: {result}!"
         if exc:
             message += "\n```\n" + exc + "\n```"
+    message += f'\nLog can be downloaded from:\nscenario-player.ci.raiden.network/{str(log_fpath).split("/var/log/scenario-player/")[1]}'
```

Comment:
I'd add `https://` to the URL.
Nitpick: Could be `log_path.relative_to(log_base_path)`